app_dart/lib/src/foundation/utils.dart

```diff
@@ -42,28 +42,28 @@ Future<List<String>> loadBranches(
 
         if (status == HttpStatus.ok) {
           final String content = await utf8.decoder.bind(clientResponse).join();
-          final List<String> branches = content.split('\n').map((String branch) => branch.trim()).toList();
-          branches.removeWhere((String branch) => branch.isEmpty);
-          return branches;
+          return content;
         } else {
-          log.warning('Attempt to download branches.txt failed (HTTP $status)');
+          log.warning('Attempt to download $filename failed (HTTP $status)');
         }
       } catch (error, stackTrace) {
-        log.error('Attempt to download branches.txt failed:\n$error\n$stackTrace');
+        log.error('Attempt to download $filename failed:\n$error\n$stackTrace');
       }
       await Future<void>.delayed(gitHubBackoffCalculator(attempt));
     }
   } finally {
     client.close(force: true);
   }
   log.error('GitHub not responding; giving up');
-  return <String>['master'];
+  return null;
```

Comment:
Isn't null the default?

Reply:
If http request is successful, then it returns the content. Otherwise it returns null.